appdev/guides/importing-CH-urldispatcher.rst

```diff
@@ -0,0 +1,154 @@
+==========================================================
+Importing from Content Hub and URLdispatcher
+==========================================================
+
+.. figure:: /_static/images/appdev/guides/importingCHdispatcherimages/01ichu.png
+        :align: center
+
+In the previous guide we have seen a little bit about how Content Hub works. In this guide we will see how URLdispatcher works and how to handle imported data from the Content Hub.
+
+Handle data from the Content Hub
+--------------------------------
+
+.. figure:: /_static/images/appdev/guides/importingCHdispatcherimages/02ichu.png
+        :align: center
+
+        OpenStore app from open-store.io
+
+One of the easiest ways of testing an app, is to send a test click to yourself, on Telegram, and opening that click file with the OpenStore through the Content Hub:
```

Comment:
No need for the second comma in this sentence.

Reply:
"One of the easiest ways of testing an app is to send a test click to yourself on Telegram, then open that click file with the OpenStore through the Content Hub"